### PR TITLE
NIFI-10578 Fix minifi-c2 unsecure integrations tests

### PR DESCRIPTION
**Pull request by briansolo1985**
# Summary

[NIFI-10578](https://issues.apache.org/jira/browse/NIFI-10578)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [x] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [x] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [x] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [x] Pull Request based on current revision of the `main` branch
- [x] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [x] Build completed using `mvn clean install -P contrib-check`
  - [x] JDK 8
  - [ ] JDK 11
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: minifi/minifi-c2/minifi-c2-integration-tests/src/test/java/org/apache/nifi/minifi/c2/integration/test/AbstractTestUnsecure.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public abstract class AbstractTestUnsecure {
     protected String c2Url;
 
     public static String getUnsecureConfigUrl(Container container) {
-        DockerPort dockerPort = container.port(10080);
+        DockerPort dockerPort = container.port(10090);
```
Comment (bejancsaba) [inline minifi/minifi-c2/minifi-c2-integration-tests/src/test/java/org/apache/nifi/minifi/c2/integration/test/AbstractTestUnsecure.java:39]: It looks like there are multiple places where 10080 is used all of those could be updated as well. Could you please take a look?

**Review by bejancsaba — CHANGES_REQUESTED** [review]
Thanks for the change however it could be extended a little further

File: minifi/minifi-c2/minifi-c2-integration-tests/src/test/java/org/apache/nifi/minifi/c2/integration/test/AbstractTestUnsecure.java (hunk 1 above)
Comment (briansolo1985) [reply minifi/minifi-c2/minifi-c2-integration-tests/src/test/java/org/apache/nifi/minifi/c2/integration/test/AbstractTestUnsecure.java:39]: Thanks for spotting this, will update

**Review by bejancsaba — APPROVED** [review]
+1 thank you

**Review by ferencerdei — APPROVED** [review]
+1

**Pull request closed without merging**